lms/djangoapps/instructor/views/api.py

```diff
@@ -2339,15 +2364,13 @@ def calculate_grades_csv(request, course_id):
     try:
         instructor_task.api.submit_calculate_grades_csv(request, course_key)
         success_status = _("The grade report is being created."
-                           " To view the status of the report, see Pending Instructor Tasks below.")
-        return JsonResponse({"status": success_status})
+                           " To view the status of the report, see Pending Tasks below.")
+        return JsonResponse({"status": success_status}, status=202)
```

Comment:
This was changed by the original PR author, so I'm not 100% sure, but it does seem to make some sense because 202 means 'Created'.